remme/shared/forms/_validators.py

```diff
@@ -88,3 +114,35 @@ def __call__(self, form, field):
 
             field.errors[:] = []
             raise validators.StopValidation(message)
+
+
+class NotRequired(object):
+    """
```

Comment:
Update documentation according to the logic.

Reply:
Done.